nexusd/config.go

```diff
@@ -22,8 +23,8 @@ type Config struct {
 	RawSocket struct {
 		// String form of address (example, "192.0.2.1:25", "[2001:db8::1]:80")
 		TCPAddress string `json:"tcp_address"`
-		// True to enable TCP keepalive
-		TCPKeepAlive bool `json:"tcp_keepalive"`
+		// Enable TCP keepalive with this interval.
+		TCPKeepAliveInterval *time.Duration `json:"tcp_keepalive_interval"`
```

Comment:
All the examples pass in `0` instead of `nil` to disable keepalive, in that case, might as well change it to `time.Duration` and remove the `!= nil` check in `nexusd/main.go`

Reply:
Done.  I do prefer to set a 0 value to disable instead of passing a pointer.